src/xcode/ENA/ENA/Source/Scenes/Onboarding/DeltaOnboarding/NewVersionFeatures/DeltaOnboardingNewVersionFeaturesViewModel.swift

```diff
@@ -81,6 +81,7 @@ struct DeltaOnboardingNewVersionFeaturesViewModel {
 	
 	private func buildNewFeaturesCells() -> [DynamicCell] {
 		var cells: [DynamicCell] = []
+		// enaFont(for style: ENAFont, weight: UIFont.Weight? = nil, italic: Bool = false)
```

Comment:
🤔 I try to understand what this comment wants to tell me, but I don't get it 😂

Reply:
LOL, mee too. Need to code cleanup this... gimme second.